bedrock/newsletter/templates/newsletter/updated.html

```diff
@@ -119,10 +119,18 @@ <h2 class="c-updated-block-header">
                 )}}
 
                 {% if switch('mozilla-monitor-brand-name') %}
+                  {% if LANG.startswith('en-') %}
```

Comment:
**Nit:** I'm not sure anymore since most files have `en-` instead of `en-US` from what the copy asks, but just noting this in case we need to change the `en-` instances to `en-US`